voidcrew/modules/more_pacmen/code/pacmen_circuitboards.dm

```diff
@@ -0,0 +1,21 @@
+// Add your pacmen circuitboards here and list the req_componets and build path
+
+/obj/item/circuitboard/machine/pacman/voidman
+	name = "VOIDMAN-type Generator"
+	greyscale_colors = CIRCUIT_COLOR_ENGINEERING
```

Comment:
assuming greyscale_colors, req_components and needs_anchored are the same here as on base pacman, it doesn't need to be here too. I think you can refer to shuttle engines for that

Reply:
Yeah i was thinking that too for a bit I will fix that up now